OpenOversight/app/static/js/find_officer.js

```diff
@@ -34,6 +34,7 @@ $(document).ready(function() {
             $('#current-uii').text(deptUiidLabel);
             $('#uii-question').show();
         } else {
+            $('#unique_internal_identifier').text('');
```

Comment:
Fixed a minor bug where unique internal identifier would still be included in the form submission if you fill in the field for a department with UII but then switched to a department without UII